src/terra-dev-site/contributing/DevelopmentPrerequisites.c.contributing.mdx

```diff
@@ -2,42 +2,27 @@ import ProgrammerIcon from '../icon/ProgrammerIcon';
+All files are released with the Apache 2.0 licence.
+Please add your name to the CONTRIBUTORS.md file in the repo you are contributing to. Adding your name to the CONTRIBUTORS.md file signifies agreement to all rights and reservations provided by the [License](https://github.com/cerner/terra-ui/blob/master/LICENSE).
 
 </div>
 <div className="tui-illustration-grid-col tui-illustration-grid-col-illustration">
 <ProgrammerIcon/>
 </div>
 </div>
 
-## Licensing
-
-All files are released with the Apache 2.0 licence.
-Please add your name to the CONTRIBUTORS.md file in the repo you are contributing to. Adding your name to the CONTRIBUTORS.md file signifies agreement to all rights and reservations provided by the [License](https://github.com/cerner/terra-ui/blob/master/LICENSE).
-
-## We Welcome PRs Regarding…
-
-* Bug fixes
-* Variants on components
-* Documentation updates
-* Increasing test coverage
-
-To help you get your feet wet and get you familiar with our contribution process, we have a list of [beginner-friendly issues](https://github.com/search?q=is%3Aopen+is%3Aissue+label%3A%22beginner-friendly%22+archived%3Afalse+repo%3Acerner%2Fterra-clinical+repo%3Acerner%2Fterra-core+repo%3Acerner%2Fterra-dev-site+repo%3Acerner%2Fterra-framework+repo%3Acerner%2Fterra-toolkit+repo%3Acerner%2Fgenerator-terra-module+repo%3Acerner%2Feslint-config-terra+repo%3Acerner%2Fstylelint-config-terra+repo%3Acerner%2Fbrowserslist-config-terra+repo%3Acerner%2Fterra-kaiju-plugin+repo%3Acerner%2Fkaiju-plugin-utils+repo%3Acerner%2Fkaiju+repo%3Acerner%2Fterra-ui+repo%3Acerner%2Fkaiju-docker-base-images+created%3A*..*) that contain bugs or small features that have a relatively limited scope. This is a great place to get started. We also have a set of issues we could use help on labelled by the "[help wanted](https://github.com/search?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22++archived%3Afalse+repo%3Acerner%2Fterra-clinical+repo%3Acerner%2Fterra-core+repo%3Acerner%2Fterra-dev-site+repo%3Acerner%2Fterra-framework+repo%3Acerner%2Fterra-toolkit+repo%3Acerner%2Fgenerator-terra-module+repo%3Acerner%2Feslint-config-terra+repo%3Acerner%2Fstylelint-config-terra+repo%3Acerner%2Fbrowserslist-config-terra+repo%3Acerner%2Fterra-kaiju-plugin+repo%3Acerner%2Fkaiju-plugin-utils+repo%3Acerner%2Fkaiju+repo%3Acerner%2Fterra-ui+repo%3Acerner%2Fkaiju-docker-base-images+created%3A*..*)" label.
-
-If you decide to fix an issue, please be sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment stating that you intend to work on it so other people don’t accidentally duplicate your effort.
-If somebody claims an issue but doesn’t follow up for more than two weeks, it’s fine to take over it but you should still leave a comment.
-
 ## Repositories
 
 Terra UI is divided into a several projects:
```

Comment:
we also have 
- terra-aggregate-translations: https://github.com/cerner/terra-aggregate-translations
- terra-enzyme-intl: https://github.com/cerner/terra-enzyme-intl

Reply:
but do i really want to advertise that?

Reply:
maybe not. They are on reference in the other docs as well though